.travis.yml

```diff
@@ -0,0 +1,7 @@
+install: true
+language: node_js
+node_js:
+  - "iojs"
+  - "7"
+
+npm install
```

Comment:
I think I figured out what our issue is. The `install: true` is causing it to run the `true` shell command instead of `npm install`. If you remove that line and the `npm install` line, Travis will automatically run npm install for us. We still need to get the token issue solved though.

One other thing, I don't think we need to support iojs, that line could be removed as well.